src/ResourceManager/Network/Commands.Network.Test/NrpTeamAlias.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
```

Comment:
@number213 please add a license header to this file

Reply:
@cormacpayne done